scripts/ci_aws_bootstrap.sh

```diff
@@ -2,76 +2,90 @@
 
 set -euo pipefail
 
-if [[ -z "${RUN_ALL:-}" ]]; then
-    RUN_ALL=0
-fi
+RUN_ALL=${RUN_ALL:-0}
+export VLLM_BUILDKITE_BRANCH="pipeline_gen"
+
+generate_pipeline() {
+    python -m pip install click pydantic
+    
+    # Download necessary files
+    for file in pipeline_generator.py plugin.py step.py utils.py; do
+        curl -o .buildkite/$file https://raw.githubusercontent.com/vllm-project/buildkite-ci/$VLLM_BUILDKITE_BRANCH/scripts/pipeline_generator/$file
```

Comment:
double quote `$FILE`, like `".buildkite/$FILE"`

Reply:
also please double quote the URL in general. URL often can contain special characters that conflicts with bash (example: `&`)